app/src/main/java/org/wikipedia/descriptions/DescriptionEditView.kt

```diff
@@ -9,16 +10,21 @@ import android.view.LayoutInflater
 import android.view.inputmethod.EditorInfo
 import android.widget.LinearLayout
 import androidx.appcompat.content.res.AppCompatResources
+import androidx.core.view.isVisible
 import androidx.core.widget.ImageViewCompat
 import androidx.core.widget.addTextChangedListener
+import de.mrapp.android.view.drawable.CircularProgressDrawable
 import org.wikipedia.R
 import org.wikipedia.WikipediaApp
+import org.wikipedia.analytics.eventplatform.MachineGeneratedArticleDescriptionsAnalyticsHelper
 import org.wikipedia.databinding.ViewDescriptionEditBinding
 import org.wikipedia.language.LanguageUtil
 import org.wikipedia.mlkit.MlKitLanguageDetector
 import org.wikipedia.page.PageTitle
+import org.wikipedia.settings.Prefs
 import org.wikipedia.suggestededits.PageSummaryForEdit
 import org.wikipedia.util.*
+import org.wikipedia.views.SuggestedArticleDescriptionsDialog
 import java.util.*
 
 class DescriptionEditView : LinearLayout, MlKitLanguageDetector.Callback {
```

Comment:
Button still appears incorrect in dark mode:

![image](https://user-images.githubusercontent.com/1682214/224100677-e7117b16-a955-4c66-ae1e-5fe8f618f601.png)